canonical-websites/www.ubuntu.com
# update /download/desktop/contribute to vbt

## Done

* update /download/desktop/contribute to vbt
* I know the button text is white... that's been fixed in several other PRs

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [page](http://0.0.0.0:8001/download/desktop/contribute?version=16.04.2&architecture=amd64)
* compare to the [design](https://www.dropbox.com/search/work?path=%2F00_web_team%2F_ubuntu.com%2Fnew+projects%2Fvanilla+update%2Fpatterns+inventory&preview=download-contribute.png&qsid=21830873026803084826401499240920&query=containers) and the [demo](http://www.ubuntu.com-1629-download-desktop-contribute.demo.haus/download/desktop/contribute?version=16.04.2&architecture=amd64)

## Issue / Card

Fixes #1629
